CHANGELOG.md

```diff
@@ -32,7 +32,7 @@
 * [BUGFIX] Ring: fix bug where instances may appear unhealthy in the hash ring web UI even though they are not. #1933
 * [BUGFIX] API: gzip is now enforced when identity encoding is explicitly rejected. #1864
 * [BUGFIX] Fix panic at startup when Mimir is running in monolithic mode and query sharding is enabled. #2036
-
+* [CHANGE] Re-enable the `proxy_url` option for receiver configuration. #2044
```

Comment:
A couple of comments about this:
1. Please link the PR number ` #2050` instead of the issue one
2. We're used to group CHANGELOG entries together, so CHANGE first, then FEATURE, then ENHANCEMENT and finally BUGFIX. See our contributing guidelines for more info: https://github.com/grafana/mimir/tree/main/docs/internal/contributing#changelog
3. CHANGE is for breaking changes, but this is not a breaking change. ENHANCEMENT looks a better fit.